routes/userRouter.js

```diff
@@ -49,8 +50,9 @@ router.post("/", async (req, res) => {
 		});
 	}
 	try {
+		const newSlug = `${spouse_one_name}and${spouse_two_name}`;
```

Comment:
It would make sense to make this a function on the user model since it's used in multiple places DRY. then a super easy unit test could be made for it. 🎉